input/pagecontent/reference.md

```diff
@@ -209,6 +209,132 @@ In cases with no explicit or logical restrictions on rule ordering, users MAY li
 
 It is possible for a user to specify contradictory rules, for example, two rules constraining the cardinality of an element to different values, or constraining an element to different data types. Implementations SHOULD detect such contradictions and issue appropriate warning or error messages.
 
+#### Indented Rules
+
+Indentation before a rule is used to set a context for the [path](#fsh-paths) on that rule. When one rule is indented below another, the full path of the indented rule is obtained by prepending the path from the rule above. For example:
+```
+* name 1..1
+  * family 1..1
+```
+is equivalent to:
+```
+* name 1..1
+* name.family 1..1
+```
+In this case, the first rule sets the "context" of `name` for any subsequent indented rules. Since the second rule is indented below the first, the `name` context is prepended to resolve the full path, `name.family`.
+
+Note that [path rules](#path-rules) can be used to set a path as context for subsequent rules, without applying any rules at that path. If in the above example, only `name.family`, but not `name`, was meant to be constrained, the rules could be specified as:
+```
+* name
+  * family 1..1
+```
+The following restrictions apply to indentation:
+* Regardless of indent, all rules must start on their own line.
+* Rules cannot be preceded by non-whitespace characters on a line.
```

Comment:
These two restrictions are applicable to all rules, not just indented rules. Does it make sense to also highlight these restrictions in a more generic section, maybe the whitespace section above?

Reply:
I went back and forth on this, it's a good point that it feels like a whitespace restriction, but that is why I linked to the rule indentation section in the whitespace section. I was wary of starting to discuss the rules around `rules` in the whitespace section, since the `Rules and Rule Order` section is sequentially later on. @markkramerus thoughts?